pytests/repo/tdnf-repoquery-queryformat.spec

```diff
@@ -0,0 +1,31 @@
+Summary:    Repoquery Test
+Name:       tdnf-repoquery-queryformat
+Version:    1.0.1
+Release:    2
+Vendor:     VMware, Inc.
+Distribution:   Photon
+License:    VMware
+Url:        http://www.vmware.com
+Group:      Applications/tdnftest
+Requires:   tdnf-test-cleanreq-required
+Conflicts:  tdnf-test-dummy-conflicts-0
+Provides:   tdnf-repoquery-queryformat
+Obsoletes:  tdnf-test-dummy-obsoleted
+
```

Comment:
This breaking other tests. Please find another way for your tests.

The `Obsoletes: tdnf-test-dummy-obsoleted` seems to break the tests `test_install_obsoleting` and `test_install_obsoleted_installed` in `test_install.py`.

The `Requires: tdnf-test-cleanreq-required` seems to break the `tdnf_py_distro_sync` test in `test_tdnf_python.py`:
```
tests/test_tdnf_python.py:231: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

quiet = True, refresh = False, cfg = None

    def tdnf_py_distro_sync(quiet=False, refresh=False, cfg=None):
        conf = cfg if cfg else config
>       tdnf.distro_sync(quiet=quiet, refresh=refresh, config=conf)
E       SystemError: <built-in function distro_sync> returned a result with an exception set

tests/test_tdnf_python.py:84: SystemError
------------------------------------------------------------------------------------ Captured stdout call -------------------------------------------------------------------------------------
Found 1 problems
tdnf-test-cleanreq-required is needed by tdnf-repoquery-queryformat-1.0.1-2.x86_64
```

There is also a failure in `test_tdnf_automatic.py` that I haven't figured out yet, but is probably somehow related too.

Reply:
Added changes to tdnf-repoquery-queryformat.spec and test_repoquery.py. Content of tdnf-repoquery-queryformat.spec is not impacting other tests

Reply:
Locally test_tdnf_python.py and test_install.py passed. test_tdnf_automatic.py is failing with and without my changes